ui/src/components/responsive_frame.jsx

```diff
@@ -59,6 +59,7 @@ const styles = {
     width: 375,
     height: 667,
     background: 'white',
-    border: '1px solid #999'
+    border: '1px solid #999',
+    overflow: 'auto'
```

Comment:
Will this conflict with https://github.com/mit-teaching-systems-lab/threeflows/pull/174/files#diff-7d9679f22d80fc6e15010306dd56b8c5L58?